package.json

```diff
@@ -121,7 +121,7 @@
     "*.js": [
       "prettier --write",
       "eslint --fix",
-      "jest --bail --findRelatedTests",
```

Comment:
Jest never finds failing snapshots because they are all in one place so just run them everytime